src/index.spec.js

```diff
@@ -361,4 +362,40 @@ describe('retryMyFetch', () => {
       });
     });
   });
+  describe('when useAbortController is turned on and set doNotRefetchIfStatuses', () => {
```

Comment:
Let's use one `describe` section for one sentence. 

There is `describe('when useAbortController is turned on', () => {` In line 51, so this describe section should be a sub section for it. 
I.e.

- when useAbortController is turned on
  - when doNotRefetchIfStatuses is set